lib/stateView/contract/mongoDbDatabase/createContractDatabaseFactory.js

```diff
@@ -0,0 +1,24 @@
+/**
+ *
+ * @param {SVDocumentMongoDbRepository} svDocumentRepository
+ * @returns {function(SVContract): Promise<*[]>}
+ */
+function createContractDatabaseFactory(svDocumentRepository) {
```

Comment:
It looks abstract. Let's move it to contract folder.

Reply:
You need to use `createSVDocumentMongoDbRepository` as a dependency and create a repository for each document type.